ns-provisioner-samples/tanzu-supply-chain/buildpack-build-clusterrolebinding.yaml

```diff
@@ -0,0 +1,14 @@
+#@ load("@ytt:data", "data")
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: #@ "buildpack-build-workload-rb-" + data.values.name
```

Comment:
Any relevance for these Rolebinding names?

Reply:
Since its is a clusterrolebinding, NSP will only create 1. So I added the namespace to the name to create 1 per namespace. It is also specific to buildpack-build component of the Supply chain. Hence the name.